app/pods/search-index/route.js

```diff
@@ -0,0 +1,10 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+    model: function (params) {
+        return this.explorer.getCluster(params.clusterId, this.store)
+            .then(function(cluster){
+                return cluster.get('searchIndexes').findBy('name', params.searchIndexId);
```

Comment:
I think I can simplify this to `cluster.findBy('searchIndexes.name', params.searchIndexId)`. Will update with other feedback received.